src/utils/emailTemplates/confirmAccountTemplate.js

```diff
@@ -0,0 +1,246 @@
+/* eslint-disable no-useless-escape */
+let firstName, token;
```

Comment:
Well done @torsami77. Good job so far. I'll just like to ask, why are you using global variables here? It is completely unavoidable? I might be wrong, but I really avoid global variables in Javascript like a plague, unless they are absolutely necessary. Is it possible to have `theEmail` be a function that accepts `firstName` and `token`, and returns that template you are already assigning to it? That way, you can call `theEmail` from `confirmAccountTemplate`, and pass it the destructured `firstName` and `token` values, instead of declaring them as global variables. Please look into this.

Reply:
Sure it is possible to avoid global variable in this case and that would trigger a more than 25 lines concern for a function. At this time am thinking it would be better to have a global variable than a 200+ lines function. Please let me have your thoughts on the issue

Reply:
Oh. Alright.